source/frontend/src/features/mapSideBar/research/ResearchContainer.tsx

```diff
@@ -285,7 +297,7 @@ export const ResearchContainer: React.FunctionComponent<
                   show
                 />
                 <ResearchView
-                  researchFile={researchFile}
+                  researchFile={researchFile as unknown as ApiGen_Concepts_ResearchFile | undefined} //todo : clean the forced upcasting
```

Comment:
were you intending to leave this in this PR unresolved?

Reply:
yes. the previous version was ok with this kind of upcasting but this one requires a forced one. We need to look deeper into how to do the casting more cleanly